docs/IE_PLUGIN_DG/Building.md

```diff
@@ -32,6 +32,7 @@ Once the commands above are executed, the Inference Engine Developer Package is
 	   * `IE::unitTestUtils` - static library with unit tests utilities 
 	   * `IE::ngraphFunctions` - static library with the set of Ngraph Functions builders
 	   * `IE::funcSharedTests` - static library with common functional tests
+	   * `IE::itt` - static library with with tools for performance measurement using Intel ITT
```

Comment:
as far as I can see, it's exported as `openvino::itt`, is not it?

Reply:
Unfortunately we can't export openvino::itt because all dependencies in ie_targets_developer.cmake should be form IE:: namespace. I added itt library to openvino_targets_developer.cmake file, but for plugins it is exported as IE::itt.